credits.txt

```diff
@@ -288,28 +316,32 @@ contributed to Endless Sky:
   thomasballinger
   ThrosturX
   Thunderforge
+  tibetiroka
   tiennou
   timbennett
   TJesionowski
   tmbutterworth
   toilethinges
+  TotalCaesar659
   TurkeyMcMac
   Turtleroku
   tux2603
-  Unjown
+  unjown
   Vilhelm16
+  vitalchip
   W1zrad
   Waffleship
-  Waladil
   warp-core
   waterhouse
+  Wedge009
   whismerhill
   willbeason
+  williaji
   wjbotham
   wjp
   Wrzlprnft
   YellowApple
-  yobbo2020
```

Comment:
Can't find any contributions here either.

Reply:
#5611, which was adapted and merged as #5683